apps/demos/Demos/DataGrid/CustomEditors/Vue/EmployeeDropDownBoxComponent.vue

```diff
@@ -57,6 +57,10 @@ const currentValue = ref(props.value);
 const dropDownBoxRef = ref<DxDropDownBox | null>(null);
 const dropDownOptions: DxDropDownBoxTypes.Properties['dropDownOptions'] = { width: 500 };
 
+const getSelectedRowKeys = (value: any): any[] => {
```

Comment:
We try to avoid use the ```any``` in our public demos and examples :)
So, let's make this method with a generic type:

```suggestion
const getSelectedRowKeys = <T>(value: T): T[] => {
```